### ISSUE TITLE
[OWF] Year 2 Launch Comms

### ISSUE THREAD

**Issue by Melechuga** [MozillaFoundation/plan#813]
#### DARCI
Driver: Melissa
A:
R: @KevZawacki @vanessarhinesmith
C: @brettgaylor 
I: @shaghdoosti @HPaulJohnson 

#### Overview
The OWF fellows launch announcement for Year 1 involved a lot of moving pieces, including websites, blogs, and media outreach. Since the launch of Year 2 will be near the time we have the "retrospective" publication ready, we might want to do something similar. 

#### Links
2015 Launch Assets:
- Copy for landing page and fellows' page: https://docs.google.com/document/d/1N_8fQ-_4GQPQIB5GdTwPw15sRVZn8pczUvkqfBb-u9Q/edit
- Fellows bios: https://docs.google.com/document/d/1_Yw4YuV1OJoe7vDS5DmIV_q8-jqeg5R4XMOLqhK7gLo/edit
- Announcement blog: https://docs.google.com/document/d/1SJiR416AkNiOXQj0AaBEBsSSTZRwpKYKysJk4fCdcc0/edit
- Comms plan, including media outreach (maybe we want to follow up about Year 2, and include the "retrospectives" publication?): https://docs.google.com/document/d/1knmXS06IKb1vcQ4wPA8W0UlZGD1JDeqznsbuBiCbG-8/edit

2016 Launch Assets:
- Copy for program description ("overview" and "info" tabs on the website) https://docs.google.com/document/d/1WRLUoMaoWvG1z_ASsQ0_WJ7NYj8E2W0Aqc6dHpZlM1U/edit

#### KPIs
Copy:
- [ ] Announce new fellows on website
- [x] Revise/compress program description ("overview" and "info" on current site)
- [ ] Blog with a little more description about fellows, selection process, host orgs.
- [ ] Whatever comms around the alumni is needed

Dependencies:
- [ ] Gather ~100word bios, photo, twitter handle from fellows

#### Key Deadlines
- [ ] Website revisions end of June
- [ ] 2016 fellows announcement early/mid July

**Comment by KevZawacki** [comment]
@Melechuga @vanessarhinesmith 

Hey -- I've outlined how I can help in a [Year 2 comms plan](https://docs.google.com/document/d/1N54hQvJToOgKV-GilquH7jsj2nDLQM6GZXRLJEX8fjo/edit).

tl;dr:

-- Assist with copywriting (website, blog, internal email)
-- Assist with channel access (Mozilla Twitter, project call)

We will message friendly policy / tech-for-good reporters as part of the plan. But, announcements like this don't get much pick-up -- so best to invest comms energy in writing, I think.

cc @HPaulJohnson

**Comment by Melechuga** [comment]
This has migrated to a google docs, and the website launched.
